Client/Frontend/Browser/BrowserViewController/BVC+Rewards.swift

```diff
@@ -230,13 +231,12 @@ extension Tab {
     }
 
     group.notify(queue: .main) {
-      let faviconURL = URL(string: self.displayFavicon?.url ?? "")
-      if faviconURL == nil {
+      if self.displayFavicon == nil {
         adsRewardsLog.warning("No favicon found in \(self) to report to rewards panel")
       }
       rewards.reportLoadedPage(
         url: url, redirectionURLs: urls.isEmpty ? [url] : urls,
-        faviconURL: faviconURL, tabId: Int(self.rewardsId),
+        faviconURL: nil, tabId: Int(self.rewardsId),
```

Comment:
We might remove the faviconURL anymore we can remove it from the function decleration